epytope/Core/Result.py

```diff
@@ -56,46 +61,61 @@ class EpitopePredictionResult(AResult):
 
         EpitopePredictionResult
 
-        +--------------+-------------+-------------+-------------+-------------+
-        | Peptide Obj  | Method Name | Allele1 Obj | Allele2 Obj | Allele3 Obj |
-        +==============+=============+=============+=============+=============+
-        | Peptide1     | Method 1    |    0.324    |     0.56    |    0.013    |
-        +              +-------------+-------------+-------------+-------------+
-        |              | Method 2    |     20      |      15     |     23      |
-        +--------------+-------------+-------------+-------------+-------------+
-        | Peptide2     | Method 1    |    0.50     |     0.36    |    0.98     |
-        +              +-------------+-------------+-------------+-------------+
-        |              | Method 2    |     26      |      10     |     50      |
-        +--------------+-------------+-------------+-------------+-------------+
+        +----------------+-------------------------------+-------------------------------+
+        |  Allele        |          Allele Obj 1         |          Allele Obj 2         | 
+        +- - - - - - - - +- - - - - - - -+- - - - - - - -+- - - - - - - -+- - - - - - - -+
+        |  Method        |    Method 1   |    Method 2   |    Method 1   |    Method 2   |
+        +- - - - - - - - +- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+
+        | ScoreType      | Score |  Rank | Score |  Rank | Score |  Rank | Score |  Rank |
+        +- - - - - - - - +- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+
+        |  Peptides      |       |       |       |       |       |       |       |       |
+        +================+=======+=======+=======+=======+=======+=======+=======+=======+
+        | Peptide Obj 1  |  0.03 |  57.4 |  0.05 |  51.1 |  0.08 |  49.4 |  0.73 |  3.12 |
+        +----------------+-------+-------+-------+-------+-------+-------+-------+-------+
+        | Peptide Obj 2  |  0.32 |  13.2 |  0.31 |  14.1 |  0.25 |  22.1 |  0.11 |  69.1 |
+        +----------------+-------+-------+-------+-------+-------+-------+-------+-------+
 
     """
 
-    def filter_result(self, expressions):
+    def filter_result(self, expressions, scoretype=False):
```

Comment:
Is the filtering and merging covered by tests?

Reply:
I tested the methods locally, but there are no unittests no..